ipfshttpclient/filescanner.py

```diff
@@ -618,11 +618,29 @@ def close(self) -> None:
 	def _join_dirs_and_files(dirnames: ty.List[AnyStr], filenames: ty.List[AnyStr]) \
 	    -> ty.Iterator[ty.Tuple[AnyStr, bool]]:
 		for dirname in dirnames:
-			yield (dirname, True)
+			yield dirname, True
 		
 		for filename in filenames:
-			yield (filename, False)
-	
+			yield filename, False
+
+	@staticmethod
+	def _walk_wide(
+			dot: AnyStr,
+			directory: ty.Union[AnyStr, int],
+			follow_symlinks: bool
+	) -> ty.Iterator[ty.Tuple[AnyStr, ty.List[AnyStr], ty.List[AnyStr], ty.Optional[int]]]:
```

Comment:
Done. A previous flavor of this method (not committed) had been using iterators.

Reply:
Using `ty.Any` means that it will be compatible with functions that expect a `SendType` other than `None` (which it is), so I think its better to use `ty.Any` since it more accurately reflects what this type is and isn't compatible with.

Reply:
Got it, added a commit to use `Any` instead of `None`